### PR TITLE
🍒 xrOS  support

### PR DESCRIPTION
**Pull request by cyndyishida**

### CODE REVIEW

**Comment by cyndyishida** [comment]
@swift-ci please test

**Comment by cyndyishida** [comment]
https://github.com/apple/swift/pull/71432
@swift-ci please smoke test

**Comment by cyndyishida** [comment]
https://github.com/apple/swift/pull/71432
@swift-ci please smoke test

**Comment by cyndyishida** [comment]
https://github.com/apple/swift/pull/71432
@swift-ci please test

**Review by JDevlieghere — APPROVED** [review]

**Comment by cyndyishida** [comment]
https://github.com/apple/swift/pull/71432
@swift-ci please test

File: clang/lib/Driver/ToolChains/Darwin.h [hunk 1]
```diff
@@ -299,6 +299,7 @@ class LLVM_LIBRARY_VISIBILITY Darwin : public MachO {
     TvOS,
     WatchOS,
     DriverKit,
+    XROS,
```
Comment (ributzka) [inline clang/lib/Driver/ToolChains/Darwin.h:302]: `LastDarwinPlatform = DriverKit` needs to be updated to `XROS`.

Comment (cyndyishida) [reply clang/lib/Driver/ToolChains/Darwin.h:302]: Thanks, will cherry-pick https://github.com/llvm/llvm-project/pull/81011

**Pull request merged**